### PR TITLE
cchsflow 1.5.0

### PR DESCRIPTION
**Pull request by wyusuf068**
New cchsflow release to be submitted to CRAN.

### CODE REVIEW

File: R/RACDPAL.R [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#' @title Participation and Activity Limitation
```
Comment (DarkShadeKnigh) [inline R/RACDPAL.R]: "Limitations"? instead of "Limitation"

File: R/RACDPAL.R [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#' 
+#' library(cchsflow)
+#' 
+#' RACDPAL_2001 <- rec_with_table(
+#'   cchs2001_p, c(
+#'     "RAC_1", "RAC_2A", "RAC_2B", "RAC_2C", "RACDPAL"
+#'   )
+#' )
+#' 
+#' head(RACDPAL_2001)
```
Comment (DarkShadeKnigh) [inline R/RACDPAL.R]: Add a non rec_with_table example to be consistent with other derived funcs

File: R/adl.R [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+#'   )
+#' )
+#' 
+#' tail(adl2009_2010)
+#' 
+#' combined_adl <- bind_rows(adl2001, adl2009_2010)
+#' 
+#' head(combined_adl)
+#' 
+#' tail(combined_adl)
```
Comment (DarkShadeKnigh) [inline R/adl.R]: Suggest adding non rec_with_table example

File: NEWS.md [hunk 4]
```diff
@@ -1,3 +1,27 @@
...
+chores
+- <font color = "green">**New**</font> ADM_RNO - Sequential record number
+- <font color = "green">**New**</font> RAC_2B - Reduction in activities at
+school or work due to disability variable used only in the 2001 CCHS survey
+cycle
+- <font color = "green">**New**</font> INCGPER_cont - continuous personal income variable
+
+## Minor improvements
+- New [article](https://big-life-lab.github.io/cchsflow/articles/duplicate_datasets.html)
+that describes overlaps in certain CCHS datasets
```
[the remark this answers is not in the document]
Comment (wyusuf068) [reply NEWS.md]: It's a bit weird but basically for a particular cycle, StatsCan has released multiple datasets. For example, the 2009-2010 CCHS survey cycle generated two PUMF data files, one that spans the two years and one that only covers 2010 respondents. The variables in both datasets differ as well.

File: R/RACDPAL.R (hunk 1 above)
Comment (wyusuf068) [reply R/RACDPAL.R]: Limitation was used across the CCHS cycles.

**Pull request merged**